go.mod

```diff
@@ -1,20 +1,35 @@
 module terraform-provider-confluent-schema-registry
 
-go 1.18
+go 1.21.0
+
+toolchain go1.21.4
 
 require (
 	github.com/ashleybill/srclient v0.6.3
 	github.com/hashicorp/go-uuid v1.0.3
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
 )
 
+require (
+	github.com/Microsoft/go-winio v0.6.2 // indirect
+	github.com/bufbuild/buf v1.38.0 // indirect
```

Comment:
This is the actual dependency I want, the rest are just related dependencies to make it work